src/app/core/messages/messages.service.spec.ts

```diff
@@ -45,7 +46,7 @@ describe('Messages Service:', () => {
 	const user = new User({ _id: new mongoose.Types.ObjectId() });
 
 	beforeEach(async () => {
-		sandbox = sinon.createSandbox();
+		sandbox = createSandbox();
```

Comment:
pretty sure this isn't even needed since nothing is being done with `sinon` in this test?

Reply:
I cleaned this up.